packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -0,0 +1,306 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  KeyCodes,
+  css,
+  getId,
+  getRTL,
+  getRTLSafeKeyCode,
+  createRef
+} from '../../Utilities';
+import { ISliderProps, ISlider } from './Slider.types';
+import { Label } from '../../Label';
+import * as stylesImport from './Slider.scss';
+const styles: any = stylesImport;
+
+export interface ISliderState {
+  value?: number;
+  renderedValue?: number;
+}
+
+export enum ValuePosition {
```

Comment:
FYI - This seems to be unused and can be deprecated / removed in a subsequent PR.

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/pull/4799